### DISCUSSION TITLE
1.4.1?

### DISCUSSION THREAD

**Discussion by apple502j** [ScratchAddons/ScratchAddons#908] [Changelog]
There are some bugs in 1.4.0 (and oops, half of it is by me refactoring the code...) - so why not have 1.4.1?

## Bugs to Fix
- #887 Addons have incorrect name. (prio-4)
- #901 Forum Search crashes. (prio-2)
- #903 Typo in Editor Messages addon description. (prio-4)
- #904 Script count was inaccurate. (prio-3)

**Comment by GrahamSH-LLK** [comment]
1.4.1 released
